src/index.js

```diff
@@ -18,19 +19,26 @@ export default class GeoBatch {
   constructor(
     {cacheFile = 'geocache.db', clientId = null, privateKey = null}
       = {cacheFile: 'geocache.db', clientId: null, privateKey: null},
+    accessor = address => address,
```

Comment:
Pass this in in the options object – why create a new param? When there are no other options, you’d have to have to pass in an empty object, what we try to prevent.